components/installer/pkg/overrides/yaml-reader.go

```diff
@@ -0,0 +1,36 @@
+package overrides
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+)
+
+const fileDir = "/kyma/installation"
+const fileName = "versions.yaml"
+
+// loadComponentsVersions reads the versions.yaml file if it exists in the /kyma/installer dir
+func loadComponentsVersions() (*bytes.Buffer, error) {
+
+	filePath := path.Join(fileDir, fileName)
+	_, err := os.Stat(filePath)
+	if err != nil {
+
+		if os.IsNotExist(err) {
+			log.Printf("File %v does not exist. Error: %v", fileName, err)
```

Comment:
If the versions.yaml is optional, maybe it should not be logged as an error that it was not created/mounted ?